program-workflows/adjust-recency-data-sync-frequency.md

```diff
@@ -0,0 +1,21 @@
+## How to Change the Frequency for Sending Recency Data to Central Server
```

Comment:
@ODORA0 How is this linked to the rest of the documentation

Reply:
@ssmusoke Well it'd help users with need to adjust sync frequencies anyway under the program-workflows